js/selectors/soknaderSelectors.js

```diff
@@ -27,3 +27,10 @@ export const hentSoknad = (state, soknad) => {
         return s.id === soknad.id;
     });
 };
+
+export const hentSoknaderSelector = (state) => {
```

Comment:
Kanskje litt forvirrende navngiving? det er en selector for utland, og selvstendig og frilanser søknader. Er tanken at den skal hente søknader som er modernisert? `modernisertSoknadSelector` kanskje?